packages/data-context/src/data/loaders.ts

```diff
@@ -28,6 +29,14 @@ export class DataLoaders {
     return this.configLoader.load(projectRoot)
   }
 
+  gitInfo (path: string) {
+    return this.gitLoader.load(path)
+  }
+
+  private gitLoader = this.loader<string, GitInfo>((absolutePaths) => {
+    return getGitInfo(absolutePaths)
```

Comment:
We'll need to account for invalidation here, but we can do that separately

Reply:
Yeah, I am guessing this is part of the general topic of "how we watch files in Cypress".